vmdb/app/models/miq_server/server_smart_proxy.rb

```diff
@@ -69,110 +69,146 @@ def vm_scan_storage_affinity
     with_relationship_type("vm_scan_storage_affinity") { children }
   end
 
-  def call_ws(ost)
```

Comment:
@lfu Please sync up with @roliveri about these changes to server_smart_proxy.  He mentioned to me that he has a branch which refactors this area and the changes will conflict.  I think we will want to drop our changes here in favor of his but we need to get some details.